frontend/src/components/shared/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -51,6 +51,29 @@ class ErrorBoundary extends React.PureComponent<Props, State> {
     const { error } = this.state
 
     if (error) {
+      if (error.name === "ChunkLoadError") {
+        return (
+          <ErrorElement
+            width={this.props.width}
+            name="Network issue"
+            message={
+              <p>
+                Please press <i>F5</i> or <i>cmd+r</i> to reload the page
```

Comment:
Use this text instead:

> Cannot load Streamlit frontend code. This can happen when you update Streamlit while a Streamlit app is running.
> 
> To fix this, simply reload this app by pressing `F5`, `Ctrl+R`, or `Cmd+R`.
> 
> If the error persists, try force-clearing your browser's cache as described [here](https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache#Cache_clearing_and_disabling).